curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md

```diff
@@ -31,7 +31,7 @@ question:
   answers:
     - |
         ```python
-        print( sorted( [ (v,k) for k,v in counts.items() ] ) )
+        print( sorted( [ (v,k) for k,v in counts.items(), reverse=True ] ) )
```

Comment:
```suggestion
        print( sorted( [ (v,k) for k,v in counts.items() ], reverse=True ) )
```

Reply:
Thanks for catching that. I feel a bit dumb...I created a small demo on repl.it to replicate the sorting, but messed up anyway.